### PR TITLE
Add slug depend on tagname

### PR DESCRIPTION
**Pull request by ArsalanSavand**

### CODE REVIEW

**Comment by amirm3hdi** [comment]
Remove the css class for overflow

File: app/entry_edit/entry_edit_controller.js [hunk 1]
```diff
@@ -21,6 +23,19 @@ function EntryEditController($scope, $state, $stateParams, $mdToast, API, Authen
    * @desc Init function for controller
    */
   function constructor() {
+
+    API.Tags.get({ tag_site: AuthenticationService.getCurrentSite() },
+      function (data, status, headers, config) {
+
+        for (var i in data.results) {
+
```
Comment (Alir3z4) [inline app/entry_edit/entry_edit_controller.js]: Remove this blank line

File: app/entry_new/entry_new_controller.js [hunk 2]
```diff
@@ -36,6 +50,16 @@ function EntryNewController($scope, $state, $mdToast, AuthenticationService, API
     ];
   };
 
+  $scope.loadTags = function (query) {
+    return load();
+  };
+
+  function load() {
```
Comment (Alir3z4) [inline app/entry_new/entry_new_controller.js]: Docs ?

File: app/entry_new/entry_new_controller.js [hunk 3]
```diff
@@ -36,6 +50,16 @@ function EntryNewController($scope, $state, $mdToast, AuthenticationService, API
     ];
   };
 
+  $scope.loadTags = function (query) {
```
Comment (Alir3z4) [inline app/entry_new/entry_new_controller.js]: This method required documentation

**Pull request merged**